packages/manager/apps/container/src/container/nav-reshuffle/sidebar/index.tsx

```diff
@@ -36,6 +37,14 @@ function Sidebar(): JSX.Element {
   const logoLink = navigationPlugin.getURL('hub', '#/');
 
   const menuClickHandler = (node) => {
+    let trackingIdComplement = 'navbar_v2_entry_';
+    const history = findPathToNode(navigationRoot, (n) => n.id === node.id)
+      .filter((item) => item.id)
+      .map((element) => element.id);
```

Comment:
i see you are building the tracking name from the ids in the sidebar, do we have some specs regarding the ids that have to be used for each menu entry or are we free to use any id we want ?